src/ops/mul_add.rs

```diff
@@ -24,13 +24,13 @@ pub trait MulAdd<A = Self, B = Self> {
     /// The resulting type after applying the fused multiply-add.
     type Output;
 
-    /// Performs the fused multiply-add operation.
+    /// Performs the fused multiply-add operation `(self * a) + b`
     fn mul_add(self, a: A, b: B) -> Self::Output;
 }
 
-/// The fused multiply-add assignment operation.
+/// The fused multiply-add assignment operation `self = (self * a) + b`
 pub trait MulAddAssign<A = Self, B = Self> {
-    /// Performs the fused multiply-add operation.
+    /// Performs the fused multiply-add assignment operation `self = (self * a) + b`
```

Comment:
Since `self` is a reference, this is really `*self = (*self * a) + b`, which you can also see in the impls below. I hope that doesn't look confusing, but if so we should probably use more words to describe it.

Reply:
Yeah I thought that pseudocode representing the mathematical operation taking place might read better, but if the other way is preferred then I'm cool with that.